GliaWidgets/Sources/View/Chat/ChatView.swift

```diff
@@ -47,7 +47,11 @@ class ChatView: EngagementView {
 
     private var heightCache: [String: CGFloat] = [:]
 
-    var props: Props
+    var props: Props {
+        didSet {
+            header.props = props.header
```

Comment:
Do you think that it would be beneficial to make a method called `renderProps` like in the other views?

Reply:
agree with @gersonnoboa, it makes sense to keep consistency and have a `renderProps` function even if it will have only one line.

Reply:
I don't mind having dedicated method for rendering props. However ChatView is not fully data driven, so for now I added  `renderHeaderProps()` because that what is really does at the moment. 2d40d7f83be93d2a6188aa8ea2074c38cdbd67f0